Services/EmailService.cs

```diff
@@ -0,0 +1,139 @@
+        {
+            _timer?.Change(Timeout.Infinite, 0);
+
+            return Task.CompletedTask;
+        }
+
+        public void Dispose()
+        {
+            _timer?.Dispose();
+        }
+
+        public void CheckUrls(object state)
+        {
+            using (var scope = _scopeFactory.CreateScope())
+            {
+                PhytimeContext _context = scope.ServiceProvider.GetRequiredService<PhytimeContext>();
+                for (int i = 0; i < _rssSource.Urls.Count; i++)
+                {
+                    XmlReader reader = XmlReader.Create(_rssSource.Urls[i]);
+                    SyndicationFeed feed = SyndicationFeed.Load(reader);
+                    reader.Close();
+                    var rssFeed = _context.Feeds.FirstOrDefault(f => f.Url == _rssSource.Urls[i]);
+                    if (rssFeed != null)
+                    {
+                        if (/*true*/rssFeed.ItemsCount != feed.Items.ToList().Count)
+                        {
+                            SendNotifications(_rssSource.Urls[i], _rssSource.Titles[i]);
+                        }
+                    }
+                    //needs to be moved to new class
+                    else
+                    {
+                        _context.Feeds.Add(new Feed { Url = _rssSource.Urls[i], ItemsCount = feed.Items.ToList().Count });
+                        _context.SaveChanges();
+                    }
+                }
+            }
+            
+        }
+
+        public void SendNotifications(string feedUrl, string feedTitle)
+        {
+            using (var scope = _scopeFactory.CreateScope())
+            {
+                PhytimeContext _context = scope.ServiceProvider.GetRequiredService<PhytimeContext>();
+                var rssfeed = _context.Feeds.Include(f => f.Users).FirstOrDefault(f => f.Url == feedUrl);
+                foreach (var user in rssfeed.Users)
+                {
+                    SendEmail(user.Email, "Обновление записей", feedTitle);
+                }
+            }
+                
+            
+        }
+
+        public void SendEmail(string email, string subject, string message)
+        {
+            var emailMessage = new MimeMessage();
+
+            emailMessage.From.Add(new MailboxAddress("Администрация сайта", "localhosttest@yandex.by"));
+            emailMessage.To.Add(new MailboxAddress("", email));
+            emailMessage.Subject = subject;
+            emailMessage.Body = new TextPart(MimeKit.Text.TextFormat.Html)
+            {
+                Text = message
+            };
+
+            using (var client = new SmtpClient())
+            {
+                client.Connect("smtp.yandex.ru", 25, false);
+                client.Authenticate("localhosttest@yandex.by", "748563akim");
+                client.Send(emailMessage);
+
+                client.Disconnect(true);
+            }
+        }
+
+        //**************need to fix*************
```

Comment:
remove comments when merging to master, just let source control manage the history